assemblies/resources/build.xml

```diff
@@ -40,12 +40,12 @@
     </delete>
   </target>
 
-  <target if="disableJobDispatching" name="job dispatching">
-    <echo>Disabling job dispatching</echo>
+  <target if="enableJobDispatching" name="job dispatching">
+    <echo>Enabling job dispatching</echo>
     <replaceregexp
-      file="target/classes/etc/org.opencastproject.serviceregistry.impl.ServiceRegistryJpaImpl.cfg"
-      match="^#dispatch.interval=2$"
-      replace="dispatch.interval=0"
+      file="target/classes/etc/org.opencastproject.serviceregistry.impl.JobDispatcher.cfg"
+      match="^# dispatch.enabled=false$"
+      replace="dispatch.enabled=true"
```

Comment:
Instead of still relying on our (admittedly well working) scripting magic, couldn't we simply have the job dispatcher in it's own module which is only deployed to nodes where it makes sense to have it?

Reply:
Extracting it into its own module would require a *lot* more reworking.  You would also still have the case where (for example) a worker node would *say* that dispatch is enabled, but wouldn't actually have the bits deployed to make it go.  IMO, this is more work than is needed, especially if we're serious about making the service registry a better high-availability system - basically a rewrite.